### PR TITLE
refactor(panel): padding and icon size adjustments

### PR DESCRIPTION
**Pull request by justinhough**
**Related Issue:** #2957

## Summary

Reverting padding and icon size.

### CODE REVIEW

File: src/components/calcite-panel/calcite-panel.scss [hunk 1]
```diff
@@ -100,7 +100,7 @@
   @apply block
     overflow-hidden
     mr-auto
-    py-5
+    py-4
     px-4;
```
Comment (macandcheese) [inline src/components/calcite-panel/calcite-panel.scss:104]: Could combine these padding items.

**Review by macandcheese — APPROVED** [review]
Lgtm 📏

File: src/components/calcite-panel/calcite-panel.scss (hunk 1 above)
Comment (justinhough) [reply src/components/calcite-panel/calcite-panel.scss]: You got it. Updated

**Review by asangma — APPROVED** [review]
Noice.

**Comment by asangma** [comment]
:shipit:

**Comment by asangma** [comment]
What the heck is going on with that screener test?

**Comment by caripizza** [comment]
@asangma I've seen that popover story fail sporadically in Screener myself (usually another run does the trick).

**Comment by asangma** [comment]
Thanks @caripizza !

**Pull request merged**